recipes/newrelic/infrastructure/ohi/nginx/linux.yml

```diff
@@ -113,27 +125,73 @@ install:
       cmds:
         - |
           stub_location=$(sudo nginx -T 2>&1 | grep -ozP "(?s:.*\s)\Klocation(?s).*stub_status" | grep -aoP "\/([^\s]+)")
+          if [ "$stub_location" == "" ]; then
+              echo -n "An nginx status stub location is required to proceed. Add default status location of /status? Y/N (default: Y)? "
+              read answer
+              echo ""
+              NEW_RELIC_CONTINUE=$(echo "${answer^^}" | cut -c1-1)
+              if [[ -z "$NEW_RELIC_CONTINUE" ]]; then
+                  NEW_RELIC_CONTINUE="Y"
+              fi
+              if [[ "$NEW_RELIC_CONTINUE" == "N" ]]; then
+                  echo "Exiting the installation code 130"
```

Comment:
Maybe a better message here? Something that indicates we stopped because the user selected not to let us add a status location that we needed. Maybe something like "Stopping installation: user opted not to allow automatic status stub page creation"

Reply:
ope, good catch, I think this was a debug print I left in. changed to your suggestion in this commit: https://github.com/newrelic/open-install-library/pull/715/commits/48ddf3b18f48c4d384a21a8e06aac72badadaae0